pkg/kotsadm/types/constants.go

```diff
@@ -27,3 +30,23 @@ func GetKotsadmLabels(additionalLabels ...map[string]string) map[string]string {
 
 	return labels
 }
+
+func GetTroubleshootLabels(additionalLabels ...map[string]string) map[string]string {
```

Comment:
The `additionalLabels` argument is not used.  Can we remove it?  Or apply the additional labels similar to `GetKotsadmLabels(...)`

Reply:
Thank you @cbodonnell. Sorry, I missed that part. Pr has been changed to use merge labels function for all additional labels. Tests also included